airbyte-integrations/connectors/source-google-adwords-singer/build.gradle

```diff
@@ -15,6 +15,12 @@ dependencies {
     implementation files(project(':airbyte-integrations:bases:base-singer').airbyteDocker.outputs)
 }
 
+// used to allow local iteration to work
```

Comment:
this code smells, why do we need it?

Reply:
Don't know, I'm not the author of this.

Reply:
I just reverted changes that were there when I took this issue.